app/scenes/channel_info/channel_info.js

```diff
@@ -135,10 +186,11 @@ export default class ChannelInfo extends PureComponent {
                         <View style={[style.separator, {backgroundColor: this.props.theme.centerChannelBg}]}/>
                     </View>
                     <ChannelInfoRow
-                        action={() => true}
+                        action={() => this.handleLeave()}
                         defaultMessage='Leave Channel'
                         icon='sign-out'
                         textId='navbar.leave'
+                        isRender={this.renderLeaveChannelRow()}
```

Comment:
Im ok with this, just the naming doesn't convince me, perhaps something like `shouldRender` ?

@hmhealey thoughts?